app/routes.py

```diff
@@ -0,0 +1,274 @@
+from flask import Blueprint, jsonify, make_response, request, abort
+from werkzeug.exceptions import NotFound
+from app import db
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from datetime import datetime, timezone
+import requests
+import os
+from werkzeug.exceptions import NotFound
+
+
+#DEFINE BLUEPRINTS
+customer_bp = Blueprint("customers", __name__, url_prefix="/customers")
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rental_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+
+#----------- HELPER FUNCTIONS -----------
+def validate_video_id(video_id):
+    try: 
+        video_id == int(video_id)
+    except:
+        abort(400, {"message": f"video {video_id} was not found"})
```

Comment:
I really like that you tell them what was wrong with the video id.  

One minor note, this isn't that the video wasn't found but that it was an invalid id.

```suggestion
        abort(400, {"message": f"{video_id} is not a valid video id"})
```